test/common/testlib.py

```diff
@@ -124,12 +139,20 @@ def __init__(self, address, label, machine, pixels_label=None, port=None):
         self.pixels_label = pixels_label
         self.machine = machine
         path = os.path.dirname(__file__)
-        self.cdp = cdp.CDP("C.utf8", verbose=opts.trace, trace=opts.trace, fixed_content_size=self.pixels_label,
+        self.cdp = cdp.CDP("C.utf8", verbose=opts.trace, trace=opts.trace,
                            inject_helpers=[os.path.join(path, "test-functions.js"), os.path.join(path, "sizzle.js")])
         self.password = "foobar"
         self.timeout_factor = int(os.getenv("TEST_TIMEOUT_FACTOR", "1"))
         self.failed_pixel_tests = 0
         self.body_clip = None
+        try:
+            with open(f'{TEST_DIR}/browser-layouts') as fp:
```

Comment:
Nitpickk: can this be named `browser-layouts.json`?

Reply:
Sure.